dbdpg/plan.sh

```diff
@@ -0,0 +1,35 @@
+pkg_name=dbdpg
+pkg_origin=core
+pkg_version="3.7.4"
+pkg_maintainer="The Habitat Maintainers humans@habitat.sh"
+pkg_license=('Artistic-1.0-Perl' 'GPL-2.0')
+pkg_deps=(
+  core/glibc
+  core/perl
+  core/postgresql
+  core/zlib
+)
+pkg_build_deps=(
+  core/cpanminus
+  core/local-lib
+  core/gcc
+  core/make
+)
+pkg_description="DBD::Pg is a Perl module that works with the DBI module to provide access to PostgreSQL databases."
+pkg_upstream_url="http://search.cpan.org/dist/DBD-Pg/"
```

Comment:
Aren't there any pkg_lib_dirs? If it's meant to be used from perl only, I'm wondering why the need for a package, instead of using cpanm to download the dep in the package that needs it? :thinking:

Reply:
for posterity:  specifying pkg_lib_dirs is for injecting things into the LD_RUN_PATH - since this is a perl module that isn't needed.